deploy/smcp-templates/base

```diff
@@ -0,0 +1,48 @@
+apiVersion: maistra.io/v1
```

Comment:
How come this file is called {{base}} and not {{base.yaml}}?

Reply:
The filenames are used as the name of the template. If the user decides to create a file named foo.yaml then the template needs to be called foo.yaml. Same with JSON. Since extension doesn't matter in Linux, I removed it to clean things up a bit.